Esri/arcgis-maps-sdk-swift-samples
# [New] Find route in mobile map package

## Description

This PR implements `Find route in mobile map package` in `Maps` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-FindRouteInMobileMapPackage/Shared/Samples/Find%20route%20in%20mobile%20map%20package)

## Linked Issue(s)

- `swift/issues/4645`

## How To Test

- Select a map from the list.
- If the map has geocoding, tap on the map to see the reverse geocode of the tap location.
- If the map has routing, tap on the map to add a point and tap again to add another with a route between them. 
- To test file importer, go to [this link](https://www.arcgis.com/home/item.html?id=260eb6535c824209964cf281766ebe43) on your simulator/device and download the map package to Files and then import it by tapping "Add Package".

## Screenshots
![find-route-in-mobile-map-package-1](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/66ca062d-8dfd-439f-b997-736492190d41) ![find-route-in-mobile-map-package-2](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/5a47c483-63c6-43d6-b279-f6103dfcd276)
